Add README.md related to run playwright tests locally (#722)
webrecorder/browsertrix: frontend/tests/README.md

# Running Tests Locally

This guide will show you how to run tests locally for a frontend application using `yarn start` and `npx playwright test`. 

## Prerequisites

Before getting started, you should have the following installed on your computer:

- Node.js
- Yarn

## Step 1: Clone the repository

Begin by cloning the repository to your local machine.

`git clone git@github.com:webrecorder/browsertrix-cloud.git`

## Step 2: Change directory to frontend

Navigate to the frontend directory:

`cd frontend`

## Step 3: Install dependencies

Install the project's dependencies:

`yarn install`

## Step 4: Add `.env`

> Add a `.env` file with the following values:
> 
> `API_BASE_URL`
> `DEV_PASSWORD`

Reviewer: We could suggest using the sample env file:
```suggestion
Add a `.env` file by copying the sample:

`cp sample.env.local .env`
```
